client/extensions/priorityHeadlineChanges/src/extension.ts

```diff
@@ -7,22 +7,21 @@ import {isEqual} from 'lodash';
 import {appConfig} from 'superdesk-core/scripts/appConfig';
 
 const extension: IExtension = {
-    id: 'priority-headline-changes',
     activate: () => {
         const result: IExtensionActivationResult = {
             contributions: {
                 authoring: {
                     onFieldChange: (fieldId, fieldsData, computeLatestEntity, exposed) => {
                         // acts as previous article here, fieldsData has the actual latest data,
                         // this one has the latest rendered data
-                        const prevArticle = computeLatestEntity();
+                        const latestRenderedArticle = computeLatestEntity();
```

Comment:
what about calling it `currentArticle`? Rendering is irrelevant in this API so I wouldn't mention it in naming either.